src/app/cmdoptions.cpp

```diff
@@ -542,6 +548,7 @@ QString makeUsage(const QString &prgName)
 #elif !defined(Q_OS_WIN)
         << DAEMON_OPTION.usage() << wrapText(QObject::tr("Run in daemon-mode (background)")) << '\n'
 #endif
+        << DEBUG_TO_LOG_OPTION.usage() << wrapText(QObject::tr("Print debug messages to qBittorrent Log")) << '\n'
```

Comment:
I would use "Redirect Qt debug messages to qBittorrent log" since on linux the messages can be enabled (to console) by env vars already.